.sync.yml

```diff
@@ -0,0 +1,15 @@
+---
+.travis.yml:
+  includes:
+  - rvm: '2.1.6'
+    sudo: required
+    services: docker
+    env: PUPPET_VERSION="3.8.4" BEAKER_set="centos-6-docker-x64"
+    script: bundle exec rake acceptance
+    bundler_args: --without development
+  - rvm: '2.1.6'
+    sudo: required
+    services: docker
+    env: PUPPET_VERSION="3.8.4" BEAKER_set="ubuntu-server-1204-docker-x64"
```

Comment:
Can you use the ~> syntax on 3.8.0 instead? Also, should we be testing on 12.04? Would 14.04 be a better target?